percona-distribution-postgresql.11/README-build.md

```diff
@@ -0,0 +1,12 @@
+
+Build image
+
+  `docker build -t percona-server-postgresql .`
+
+Tag image
+  
+  `docker tag <NNNNN> percona/percona-server-postgresql:11`
```

Comment:
please replace percona with perconalab (everywhere )as for now image will go into perconalab

Reply:
And with percona-distribution-postgresql instead of percona-server

Reply:
Also updated README for both packages